src/main/java/com/budgetapp/controller/UserController.java

```diff
@@ -0,0 +1,32 @@
+package com.budgetapp.controller;
+
+import com.budgetapp.entities.User;
+import com.budgetapp.services.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class UserController {
+private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping("/user")
```

Comment:
Following RESTful API design, you want to make sure your resource paths are plural

```suggestion
    @GetMapping("/users")
```